lib/bungee/src/Stretch.cpp

```
@@ -0,0 +1,39 @@
+
+	multiplier = (int32_t)std::max<float>(std::round((1 << shift) / -speed), std::numeric_limits<int16_t>::min());
+	BUNGEE_ASSERT1(multiplier <= 0);
+}
+
+void Frequency::operator()(int n, Eigen::Ref<Eigen::ArrayX<Phase::Type>> rotation, const Eigen::Ref<Eigen::ArrayX<Phase::Type>> &phase) const
+{
+	rotation[0] = 0;
+	for (int m = 1; m < n; ++m)
+	{
+		Phase::Type delta = phase[m - 1] - phase[m];
+
```

Cppcheck (2.19.0)

[style] The function 'cast' is never used.

(unusedFunction)